lib/api/src/grpc/proto/collections.proto

```diff
@@ -326,13 +326,13 @@ message PayloadSchemaInfo {
 message CollectionInfo {
   CollectionStatus status = 1; // operating condition of the collection
   OptimizerStatus optimizer_status = 2; // status of collection optimizers
-  uint64 vectors_count = 3; // number of vectors in the collection
+  optional uint64 vectors_count = 3; // number of vectors in the collection
```

Comment:
Is it backward compatible? I find it untrivial in case of backward compatibility